Xamarin.Forms.Material.iOS/MaterialEditorRenderer.cs

```diff
@@ -0,0 +1,116 @@
+﻿using CoreGraphics;
+using UIKit;
+using MaterialComponents;
+using System.Threading.Tasks;
+using Foundation;
+using System;
+
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialEditorRenderer : EditorRendererBase<MaterialMultilineTextField>, IMaterialEntryRenderer
+	{
+		bool _hackHasRan = false;
```

Comment:
we'll regret this one day

Reply:
Once we update to latest iOS Material Components we can remove this code